MultiSigWalletOverride/truffle.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
```

Comment:
Do we need to deploy those Multisig contracts to other networks too?? Or we reuse their existing ones (if there are such ones)?

Reply:
aha, yeah, we can add as many networks as we want!